AnimeXNovel/dist/presence.js

```diff
@@ -0,0 +1,62 @@

```

Comment:
You cant have sourcemapping if you only have a js file, so upload the ts files or remove it.
I suggest you to upload the ts files

Reply:
I removed the sourcemapping, because i don't know TS